AddLicensesToFiles.py

```diff
@@ -23,7 +23,7 @@ def update_copyrights_glob(globpattern, copyright_string):
 def update_copyright_file(filepath, copyright_string):
     global update_count
     copyright_lines = copyright_string.split("\n")
-    lines = open(filepath).readlines()
+    lines = open(filepath, errors='ignore').readlines()
```

Comment:
Did you hit an issue that required this?

Reply:
Yes, when it got to `\mantidimaging\core\io\test\filenames_test.py`, there are emojis in some of the filenames that `open()` could not decode. I tried a few encoding options but couldn't find one that left the file as it was before so I changed the file manually instead. The `errors='ignore'` is there so that all the other files were processed as normal.